index.js

```diff
@@ -147,6 +152,8 @@ module.exports = class ServeDrive extends ReadyResource {
   }
 
   async _onrequest (req, res) {
+    if (this.closing) return
```

Comment:
dont think this one is needed as nothing async has happened before (no controlflow givne up)

Reply:
Yeah, wasn't sure here since this is the entrypoint of the on('request') listener, so the entrypoint itself is async in a sense. Is this safe against the following scenario?
- serveDrive.close() (ready resource method) is called => `serveDrive._close()` is scheduled
- Some new requests come in and are scheduled
- `serveDrive._close() is executed, so all open connections are destroyed, and the `.close()` of the http server is started
- The scheduled request handlers start executing. If the req is invalid (not GET or HEAD), it will try to reply with a 400 status on a destroyed connection
- the http servers `close()` finishes

If so, can indeed be removed

Reply:
a new request shouldn't come in, we are calling server.close() sync in the close handler

Reply:
Okay, removed that one